crypto/mem_sec.c

```diff
@@ -356,6 +357,8 @@ static int sh_init(size_t size, int minsize)
     sh.minsize = minsize;
     sh.bittable_size = (sh.arena_size / sh.minsize) * 2;
 
+    if (sh.bittable_size > INT_MAX)
+      goto err;
```

Comment:
because bittable_size is a size_t, unsigned int, and other parts of the code treat dependant data structures as if things are int-sized.